tests/s3/copy_object/test_parallel_copy_object.py

```diff
@@ -63,8 +68,19 @@ def setup_method(self):
         LOGGER.info("STARTED: Test setup.")
         if not os.path.exists(self.test_dir_path):
             os.makedirs(self.test_dir_path, exist_ok=True)
-        self.buckets = ["src-bucket", "dest-bucket", "src-bucket1", "src-bucket2", "dest-bucket1",
-                        "dest-bucket2"]
+        self.src_bkt = "src-bucket"
```

Comment:
would be a good idea to randomize the names a bit, applicable for this line till line 81. You can update them to have a suffix with current time - 
```suggestion
        self.src_bkt = "{0}{1}".format("src-bucket", time.perf_counter_ns())
```